.travis.yml

```diff
@@ -1,8 +1,11 @@
 language: go
 go:
-  - 1.6.x
   - 1.7.x
   - 1.8.x
   - master
 
+matrix:
+ allow_failures:
+   - go: master
```

Comment:
You might want to enable `fast_finish` here as well, so build status is reported more quickly.

See https://github.com/shurcooL/home/blob/e6f2e71819d2db3cd6fb1a781ac880cbfff3ed9b/.travis.yml#L9 for example.

See https://blog.travis-ci.com/2013-11-27-fast-finishing-builds/ for reference.